content/blog/hpe-greenlake-for-private-cloud-enterprise-glpce-deploying-and-scaling-traditional-applications.md

```diff
@@ -0,0 +1,97 @@
+---
+title: HPE GreenLake for Private Cloud Enterprise (GLPCE) Deploying and scaling
+  traditional applications
+date: 2023-10-03T03:45:24.053Z
+author: Sudheendra Seshachala Rao
+authorimage: /img/1695127848528.jpeg
+disable: false
+---
```

Comment:
Please add the tag:
hpe-greenlake-for-private-cloud-enterprise
to tie your blog post to the HPE GreenLake for private cloud enterprise landing page on the HPE Dev portal
https://developer.hpe.com/greenlake/hpe-greenlake-for-private-cloud-enterprise/home/

Reply:
Done